dev-infrastructure/templates/svc-cluster.bicep

```diff
@@ -371,60 +371,60 @@ module imageServiceKeyVaultAccess '../modules/keyvault/keyvault-secret-access.bi
-      grantPushAccess: true
-      acrResourceGroupid: imageSyncAcrResourceGroups[i].id
-    }
-  }
-]
-
-resource clustersServiceAcrResourceGroups 'Microsoft.Resources/resourceGroups@2023-07-01' existing = [
-  for rg in clustersServiceAcrResourceGroupNames: {
-    name: rg
-    scope: subscription()
-  }
-]
-
-module acrManageTokenRole '../modules/acr-permissions.bicep' = [
-  for (_, i) in clustersServiceAcrResourceGroupNames: {
-    name: guid(clustersServiceAcrResourceGroups[i].id, resourceGroup().name, 'clusters-service', 'manage-tokens')
-    scope: clustersServiceAcrResourceGroups[i]
-    params: {
-      principalId: csManagedIdentityPrincipalId
-      grantManageTokenAccess: true
-      acrResourceGroupid: clustersServiceAcrResourceGroups[i].id
-    }
-  }
-]
-
-// oidc
-
-module oidc '../modules/oidc/main.bicep' = {
-  name: '${deployment().name}-oidc'
-  params: {
-    location: location
-    storageAccountName: oidcStorageAccountName
-    rpMsiName: clusterServiceMIName
-    skuName: oidcStorageAccountSku
-    aroDevopsMsiId: aroDevopsMsiId
-    deploymentScriptLocation: location
-  }
-  dependsOn: [
-    svcCluster
-  ]
-}
+// resource imageSyncAcrResourceGroups 'Microsoft.Resources/resourceGroups@2023-07-01' existing = [
+//   for rg in imageSyncAcrResourceGroupNames: {
+//     name: rg
+//     scope: subscription()
+//   }
+// ]
+
+// module acrPushRole '../modules/acr-permissions.bicep' = [
+//   for (_, i) in imageSyncAcrResourceGroupNames: {
+//     name: guid(imageSyncAcrResourceGroups[i].id, resourceGroup().name, 'image-sync', 'push')
+//     scope: imageSyncAcrResourceGroups[i]
+//     params: {
+//       principalId: imageSyncManagedIdentityPrincipalId
+//       grantPushAccess: true
+//       acrResourceGroupid: imageSyncAcrResourceGroups[i].id
+//     }
+//   }
+// ]
+
+// resource clustersServiceAcrResourceGroups 'Microsoft.Resources/resourceGroups@2023-07-01' existing = [
+//   for rg in clustersServiceAcrResourceGroupNames: {
+//     name: rg
+//     scope: subscription()
+//   }
+// ]
+
+// module acrManageTokenRole '../modules/acr-permissions.bicep' = [
+//   for (_, i) in clustersServiceAcrResourceGroupNames: {
+//     name: guid(clustersServiceAcrResourceGroups[i].id, resourceGroup().name, 'clusters-service', 'manage-tokens')
+//     scope: clustersServiceAcrResourceGroups[i]
+//     params: {
+//       principalId: csManagedIdentityPrincipalId
+//       grantManageTokenAccess: true
+//       acrResourceGroupid: clustersServiceAcrResourceGroups[i].id
+//     }
+//   }
+// ]
+
+// // oidc
+
+// module oidc '../modules/oidc/main.bicep' = {
+//   name: '${deployment().name}-oidc'
+//   params: {
+//     location: location
+//     storageAccountName: oidcStorageAccountName
+//     rpMsiName: clusterServiceMIName
+//     skuName: oidcStorageAccountSku
+//     aroDevopsMsiId: aroDevopsMsiId
+//     deploymentScriptLocation: location
+//   }
+//   dependsOn: [
+//     svcCluster
+//   ]
+// }
```

Comment:
the permission issue has been fixed in main - https://github.com/Azure/ARO-HCP/pull/897